src/mailing/sender.js

```diff
@@ -9,26 +11,28 @@ const transport = nodemailer.createTransport({
   },
 });
 
-// Function to send a test email (promisified)
-async function sendTestEmail(recipient) {
-  return new Promise((resolve, reject) => {
-    transport.sendMail(
-      {
-        from: process.env.EMAIL_USERNAME,
-        to: recipient,
-        subject: 'Test Email',
-        text: 'Hello, this is a test email!',
-        html: '<p>Hello, this is a test email!</p>',
-      },
-      (err, res) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(res);
-        }
-      }
-    );
+const baseEmail = new Email({
+  message: {
+    from: 'codethedream.practicum.team4@outlook.com',
+  },
+  transport: transport, // Use the transport you created
+  subjectPrefix: process.env.NODE_ENV === 'production' ? '' : '(TEST) ',
+  views: {
+    root: path.resolve('../templates/test'), // this tells email-templates to look for templates starting from this path
+    options: {
+      extension: 'ejs', // this tells email-templates that the file will end tith *.ejs
+    },
+  },
+});
+
+// Function to send a test email
+async function sendTestEmail(to, username) {
```

Comment:
To be able to test this, you'll want to update `src/mailing/test.js` file to pass in a `"username"` to be able to see it in the preview email.

Reply:
@akosasante I´ve hardcoded username- maybe it would be better to add here a username from the User model- can I do that and import it from the User.js ?  I´ve hardcoded username- maybe it would be better to add here a username from the database - from 
User model- can I do that and import it from the User.js ?

Reply:
I will try it now

Reply:
current code doesn´t work